src/sql/connection/connection.py

```diff
@@ -140,7 +141,15 @@ class ConnectionManager:
     current = None
 
     @classmethod
-    def set(cls, descriptor, displaycon, connect_args=None, creator=None, alias=None):
+    def set(
+        cls,
+        descriptor,
+        displaycon,
+        connect_args=None,
+        creator=None,
+        alias=None,
+        config=None,
+    ):
```

Comment:
what params can be passed in `config`?

Reply:
added docstring. note that I only documented some of the arguments because some are legacy and I'm not even sure what they're used for